finality-aleph/src/network/session/service.rs

```diff
@@ -319,7 +319,7 @@ where
                     pen,
                 };
                 let (actions, data_from_network) =
-                    self.manager.update_validator_session(pre_session).await?;
+                    self.manager.update_validator_session(pre_session)?;
```

Comment:
So this function doesn't have to be `async` either now, does it?

Reply:
good catch, done